scripts/apps/authoring/authoring/helpers.js

```diff
@@ -120,6 +120,8 @@ export function stripHtml(item) {
             item[key] = stripHtmlRaw(item[key]);
         }
     });
+
```

Comment:
Looks like this function is no longer relevant since we no longer output HTML for the headline?

Reply:
It's not but I would keep the `return` statement as it's an expected behaviour for a function. Maybe this should be a separate commit. Also we should not modifiy the input but create a new item instead.

Reply:
Judging from the fact that `fields` is hardcoded to `['headline']` I think this function can be removed altogether.